test/testInjectScripts.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const assert = require('assert');
+const fs = require('fs');
+const tmp = require('tmp');
+tmp.setGracefulCleanup();
+
+const {injectScriptsIntoHTML} = require('../src/injectScripts.js');
+
+
+describe('#injectScriptsIntoHTML', function() {
+  context('Inject 2 Scripts', function() {
+    const tmpFile = tmp.fileSync();
+    const testFile = './test/testdata/input/file-list-test.html';
+    const scripts = [];
+    scripts.push(fs.readFileSync('./test/testdata/reference/transformed_script0_file-list-test.js', 'utf-8'));
```

Comment:
undecided what's best, but throwing this out there - instead of having these two js files, can we have raw strings at the top of this file with the file contents? One benefit is seeing everything in one place and it matching more closely with the new APIs. Makes this code a bit uglier though

Reply:
I was undecided as well. I thought if we have any intention (which I did) of adding more tests, that method would get unruly really fast, making it harder to understand and view the test file. But I do understand the pain of having to dig up the reference files to understand the internals of the test. Not really sure what to do...

Reply:
Only solution that comes to mind is to have a single raw string that covers the variety of things that the unit tests cover. The unit tests would need to be selective about which things they check, though, which probably introduces a bunch of other complexity.

So since neither of us hates this, lets leave as-is :)

Reply:
FWIW I slightly prefer having them as files :)